common/app/slices/CommercialContainer.scala

```diff
@@ -0,0 +1,13 @@
+package slices
+
+object CommercialContainer {
+  val all: Map[String, Container] = Map(
+    ("commercial/single-campaign", Commercial(SingleCampaign(FixedContainers.fixedSmallSlowIV))),
+    ("commercial/multi-campaign", Commercial(MultiCampaign(FixedContainers.fixedSmallSlowIV)))
+  )
+}
+
+sealed trait CommercialContainer
```

Comment:
Not so sure about this name... Maybe `CommercialCampaign`?
The thing on the outside of this is the `Container`, while the thing inside is the `ContainerDefinition`. This middle part has never exited before but I don't think it's strictly containter!